library/Claim.class.php

```diff
@@ -1079,6 +1080,12 @@ function box14qualifier()
       return empty($this->billing_options['box_14_date_qual']) ? '431' :
               $this->billing_options['box_14_date_qual'];
   }
+
+  function en_pos_code()
+  {
```

Comment:
This appears to subvert the purpose of the Claim class.  It should have one function that returns the POS code to be used for the claim.  The other modules that generate claims in X12 or paper format should not have to concern themselves with where that data came from.  Keep the "business logic" here to the degree that is possible.

Also let's be consistent with the "camelCase" convention for function names.